docs/001-devices.apib

```diff
@@ -307,3 +307,126 @@ Returns a single IP address object.
 Unassign an IP address record. This will remove the relationship between an IP and the device and will make the IP address available to be assigned to another device.
 
 + Response 204
+
+## Traffic [/devices/{id}/traffic{?direction,timeframe,interval,bucket}]
+Get traffic for a specific device.
+
++ Parameters
+    + id (string) - The device ID in the form of a UUID
+    + direction (enum[string]) - Traffic direction
+        + Members
+            + inbound
+            + outbound
+    + timeframe (date, required) ... Date UTC format.
```

Comment:
> timeframe (date, required) ... Date UTC format.
> +      It is an array with `started_at` and `ended_at` keys to set the dates between what you wanted to get the traffic.

This seems confusing--we are saying both that it's a `date` and an `array`. Also, the sentence would sound better as something like the following:

> An array with `started_at` and `ended_at` keys.